core/templates/dev/head/components/forms/schema-based-editors/schema-based-float-editor.directive.ts

```diff
@@ -16,7 +16,13 @@
  * @fileoverview Directive for a schema-based editor for floats.
  */
 
+require(
+  'components/forms/custom-forms-directives/apply-validation.directive.ts');
+require(
+  'components/forms/custom-forms-directives/require-is-float.directive.ts');
```

Comment:
Should there be empty line after this line? Because :arrow_double_up: are used in the HTML?